docs/sample_config.yaml

```diff
@@ -1352,3 +1352,18 @@ password_config:
 #  module: "my_custom_project.SuperRulesSet"
 #  config:
 #    example_option: 'things'
+
+
+## Tracer ##
+
+#tracer:
```

Comment:
I'd call this `opentracing` to be a bit more explicit. Please also add a few sentences to say what this config stuff does (i.e. "this enables opentracing which allows...")